firebase-firestore/src/main/java/com/google/firebase/firestore/DocumentReference.java

```diff
@@ -71,6 +72,7 @@
   }
 
   /** @hide */
+  @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
We recently dropped the 'public' from this, so we don't need to explicitly "RestrictTo" it. i.e. if you merge master, you can revert your changes to this file.

Reply:
Merged and removed.